.travis.yml

```diff
@@ -29,6 +29,3 @@ script:
   - make test.everything
 after_success:
   - codeclimate-test-reporter
-notifications:
-    slack:
```

Comment:
Didn't we decide to leave the slack notifications for Travis, but for failure only, and send to the alerts channel?

Reply:
Ahh, yes! I can change that before I merge